src/services/role/utils/unique.js

```diff
@@ -0,0 +1,3 @@
+const unique = (...permissions) => ([...new Set(Array.prototype.concat.apply([], permissions))]);
```

Comment:
```suggestion
const unique = (...permissions) => ([...new Set(Array.from(permissions))]);
```

Kann gerne auch weiter hoch in der Hierarchie; sonst benutzt es niemand anderes.

Reply:
In utils, dann würde ich den input noch umbenennen.